home_work/hw_11/bank_deposit/currency.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from loguru import logger as log
+
+API_KEY = '1240d98b1bc1385eea7ad602'
+API_URL = f"https://v6.exchangerate-api.com/v6/{API_KEY}/latest/USD"
```

Comment:
ok, let's do it more flexible:

```suggestion
base_url = "https://v6.exchangerate-api.com/v6/"
currency = "USD" | "EUR" | "BYN" etc
URL = f"{base_url}{API_KEY}/{currency}"
```

What is more, based on their [doc](https://www.exchangerate-api.com/docs/authentication), "Token Bearer Authentication" is possible to use, it is even better than sending a token in the URL:

```suggestion
base_url = "https://v6.exchangerate-api.com/v6/latest/"
currency = "USD" | "EUR" | "BYN" etc
URL = f"{base_url}{currency}"
```

and use token in auth headers, like:
```suggestion
headers = {
  'Content-Type': 'application/json',
  'Authorization': '••••••'
}

response = requests.get(URL, headers=headers)
print(response.text)
```

Reply:
@Stml89 nice, now all secrets keeps in .env and key sending in headers. nothing was broken. One question, how to correctly store .env if i want to share secret with my develop team ?

Reply:
`.env` file is used for development porpoise.
Store them in the secrets manager, ansible or [this](https://medium.com/@c17hawke/unlock-secrets-to-managing-your-credentials-with-python-dotenv-quickstart-guide-5e0e34cd5c9a), or [this](https://medium.com/@fridahkimathi/securing-credentials-in-python-59d2092d7b64)